src/Analyzer.Cli.UnitTests/AppServicesLogs-Failures.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Is there any way we can refer to this file without duplicating it in the repo? Do you need the entire file or can you pass a skeleton or smaller version of it? Same with the passes.json.

Reply:
Error on my part, I shortened the "Passes" and forgot to do the same with the "Failures"